### PR TITLE
added vite server to package.json to solve installation issue

### PR DESCRIPTION
**Pull request by jamiewaese**
I had to install vite separately in order to run this. I'm not sure why it wasn't in the package.json file. Anyway, now it is!

### CODE REVIEW

**Comment by asherpasha** [comment]
Thanks. Please let me know when it is ready to merge.

**Comment by jamiewaese** [comment]
It’s just adding a line to the package.json. Should be safe.

**Review by alexwaeseperlman — CHANGES_REQUESTED** [review]
It looks like you also committed some design changes to this branch. Do you want to open another pull request?

**Comment by jamiewaese** [comment]
Not yet, I need to fix something I broke first.

Asher, better not do anything yet. I'll let you know when it's ready.

File: Eplant/UI/Layout/ViewContainer/LoadingPage.tsx [hunk 1]
```diff
@@ -64,16 +65,17 @@ export default function LoadingPage(props: {
   return (
     <Stack gap={4}>
       <Typography variant="body2">
-        Loading data for {props.view.name}...
+        Loading {props.gene.id + ' data for ' + props.view.name}
```
Comment (alexwaeseperlman) [inline Eplant/UI/Layout/ViewContainer/LoadingPage.tsx:68]: This needs to handle the case where `props.gene` is null.

**Pull request merged**